### ISSUE TITLE
Automatically generated alt text is problematic for screen reader users

### ISSUE THREAD

**Issue by bravegrape** [middleman/middleman#2319]
## Background

Back in 2013, the Middleman community had the best intentions when building the [automatic additon of the alt-attribute](https://github.com/middleman/middleman/issues/782), inspired by the Rails community.

Automatically generating alt text based on file names is not a good idea because:

* file names are not good alt text
* this is a feature users do not expect

A digital accessibility specialist also [pointed out similar concerns to the Rails community](https://github.com/rails/rails/issues/30096) who then removed the feature.

### File names are not good alt text

Automatically generated alt text based on the file name is problematic for screen reader users. This is because the file name is never a contextualised description of the image it contains. For example, a file called `infrastructure-overview.svg` says nothing about the infrastructure itself.

### Users do not expect automatic addition of alt text

A lot of Middleman users are unaware of this feature that automatically generates alt text for them. For example, my team needed to add [a decorative image which should have an empty alt tag](https://www.w3.org/WAI/tutorials/images/decision-tree/). We thought that adding an image using `![](images/logo.svg)` would produce an empty alt tag. It wasn't until much later when we looked at the HTML for an unrelated reason that we noticed the automatically generated alt tag.

## Steps to reproduce the problem (from a clean middleman installation)

Adding an image to a Middleman site using both `![](images/logo.svg)` and `<%= image_tag "logo.svg" %>` produces:

```html
<img src="/images/logo.svg" alt="Logo">
```

This is not the expected outcome for users who are not aware of the automatically generated alt text.

## Additional information

- Ruby version: 2.6.5
- Middleman version: 4.3.6
- OS version: MacOS 10.12.6
